responsive-web-design/Build-product-landing-page/creative.css

```diff
@@ -0,0 +1,311 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+}
+
+body {
+    /* background-color: #fda635; */
+    background-image: linear-gradient( 115deg, #ed4b4bba, #285da2a6, #834343), url(https://thewindow.barneys.com/wp-content/uploads/2017/11/4600_readyforretouching.jpg);
```

Comment:
Why are we using an external URL for this image??

If for some reason the image on the page is deleted your website going to be affected, please download the image and include it into this project locally.